TombEngine/Game/control/box.cpp

```diff
@@ -722,12 +726,32 @@ short CreatureTurn(ItemInfo* item, short maxTurn)
 	return angle;
 }
 
+static void PlayGunEffectForCreature(ItemInfo* item, const CreatureMuzzleflashInfo& muzzleFlash)
```

Comment:
Why `static`?

Reply:
It's not used elsewhere and i think it won't be.